### ISSUE TITLE
ediola V6 with Hex

### ISSUE THREAD

**Issue by wocowahl** [andyboeh/mediola2mqtt#23]
Hi,
 First of all thanks for the nice integration!
 I am running Mediola V6 and I added Elero Blind adr 01-14. 01-09 are working; but 10-14 are not.
 Extract:

 *- type: ER
   adr: "08"
 - type: ER
   adr: "09"
 - type: ER
   adr: "10"
 - type: ER
   adr: "11"
 *
 Via /command?XC_FNC=GetStates I already got:
 *{"type":"ER","sid":"10","adr":"14","state":"1002","ts":{"a":"63541871"}}*
 which I tranferred to

 *- type: ER
   adr: "14"
 *
 The log does not provide additional information.
 E.g. working adr 9 compared to not working adr 11:

 *Received PUBLISH (d0, q0, r0, m0), 'mediola/blinds/mediola/ER_09/set', ...  (4 bytes)
 Msg: mediola/blinds/mediola/ER_09/set 0 b'open'
 Received PUBLISH (d0, q0, r0, m0), 'mediola/blinds/mediola/ER_11/set', ...  (5 bytes)
 Msg: mediola/blinds/mediola/ER_11/set 0 b'close'
 *
 Many thanks for your ideas,
 Wolfgang

**Comment by rorso** [comment]
Count the channels in Hex 0A - 0E and you may leave out the delimiters, as the channel is numeric anyway.

23.10.2022 08:15:54 wocowahl ***@***.***>:

**Comment by andyboeh** [comment]
Thank you, that finally means: another difference between v4 and v6 (v5?). On my v4, all channels are in hex while on yours they are in dec. 

That will be a bigger change, unfortunately including backwards-incompatible changes. I'm planning a bigger rewrite for quite some time now, looks like I should get started soon. (Unfortunately, ) real life is keeping me busy right now.

**Comment by wocowahl** [comment]
hi,
I am new in that topic. I recognized that #11 pretty much deals with my topic, but I was no longer able to delete my created issue.
anyway for sharing I changed as suggested in https://github.com/andyboeh/mediola2mqtt/issues/11#issuecomment-1128157716 and for me it is working.
So my config is in dec and there is no converting at all (e.g. `adr: "14"`).
I do not use any `.upper()`.


many thanks,
Wolfgang
